src/Bicep.Core/Syntax/ProviderDeclarationSyntax.cs

```diff
@@ -33,20 +36,40 @@ public ProviderDeclarationSyntax(IEnumerable<SyntaxBase> leadingNodes, Token key
 
         public SyntaxBase AsClause { get; }
 
-        public ImportSpecification Specification => lazySpecification.Value;
+        public IProviderSpecificationSyntax Specification => lazySpecification.Value;
 
         public ObjectSyntax? Config => (this.WithClause as ProviderWithClauseSyntax)?.Config as ObjectSyntax;
 
         public IdentifierSyntax? Alias => (this.AsClause as AliasAsClauseSyntax)?.Alias;
 
         public override TextSpan Span => TextSpan.Between(this.Keyword, TextSpan.LastNonNull(this.SpecificationString, this.WithClause, this.AsClause));
 
-        SyntaxBase IArtifactReferenceSyntax.SourceSyntax => SpecificationString;
+        public SyntaxBase SourceSyntax => SpecificationString;
```

Comment:
Why change this?

Reply:
Both expressions are equivalent, in other places in the code where we implement an interface property, we don't qualify it as above and I had introduced this change in the past so why should we specifically do it here?

Reply:
They aren't equivalent: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation.

Reply:
Thank you so much for sharing the docs to help educate me on this. It appears that an explicit interface implementation is helpful when the author is interested in distinguishing implementations of a function whose signature is matching for two separate interfaces that the object implements.

This is not the case here, the getter for `SourceSyntax` is only defined in `IArtifactReferenceSyntax` and the intent here is to implement that function. No other inherited types have such a function definition so there is no need to use an explicit interface.